exercise1/submissions/README.md

```diff
@@ -0,0 +1,21 @@
+Exercise 1 Notes
+================
+
```

Comment:
Thanks for adding README.md file. It's a small but really thoughtful effort in documentation.